From e1658eafc79e47e9ba999a9e52689fc7ba7363e3 Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Thu, 28 Aug 2014 14:24:39 -0400
Subject: [cpufreq] Various RHEL cleanups

Message-id: <1409235879-1283-186-git-send-email-prarit@redhat.com>
Patchwork-id: 89441
O-Subject: [RHEL7.1 PATCH BZ 1134639 185/185] Various RHEL cleanups
Bugzilla: 1134369
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>
RH-Acked-by: Mark Langsdorf <mlangsdo@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1134639

drivers/cpufreq/cpufreq.c
 __cpufreq_add_dev() has a two white space errors introduced by
 RHEL commit cc3d3b99f3a290188c114210fb5e77aee92ff0a0
drivers/cpufreq/cpufreq_stats.c
 Fix extra space introduced by
 RHEL commit 78b6514695f881a405d25626a9bcc1c5e1dd91a1

Remove CONFIG_CPU_FREQ_TABLE which is no longer needed.

RHEL_only. (obviously)

Cc: Lenny Szubowicz <lszubowi@redhat.com>
---
 delete mode 100644 redhat/configs/generic/CONFIG_CPU_FREQ_TABLE

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/cpufreq/cpufreq.c b/drivers/cpufreq/cpufreq.c
index afa4109..d9fdedd 100644
--- a/drivers/cpufreq/cpufreq.c
+++ b/drivers/cpufreq/cpufreq.c
@@ -1277,6 +1277,7 @@ static int __cpufreq_add_dev(struct device *dev, struct subsys_interface *sif)
 
  kobject_uevent(&policy->kobj, KOBJ_ADD);
  up_read(&cpufreq_rwsem);
+
  pr_debug("initialization complete\n");
 
  return 0;
@@ -1300,6 +1301,7 @@ err_set_policy_cpu:
 
 nomem_out:
  up_read(&cpufreq_rwsem);
+
  return ret;
 }
 
diff --git a/drivers/cpufreq/cpufreq_stats.c b/drivers/cpufreq/cpufreq_stats.c
index bb72188..0cd9b4d 100644
--- a/drivers/cpufreq/cpufreq_stats.c
+++ b/drivers/cpufreq/cpufreq_stats.c
@@ -15,7 +15,6 @@
 #include <linux/slab.h>
 #include <linux/cputime.h>
 
-
 static spinlock_t cpufreq_stats_lock;
 
 struct cpufreq_stats {
diff --git a/redhat/configs/generic/CONFIG_CPU_FREQ_TABLE b/redhat/configs/generic/CONFIG_CPU_FREQ_TABLE
deleted file mode 100644
index 95997ad..0000000
--- a/redhat/configs/generic/CONFIG_CPU_FREQ_TABLE
+++ /dev/null
@@ -1 +0,0 @@
-CONFIG_CPU_FREQ_TABLE=y
-- 
1.7.1